Description: Add missing closing bracket
 On i386, compilation fails because of this.
Author: Frederic Bonnard <frediz@linux.vnet.ibm.com>
Bug: https://github.com/sphde/sphde/pull/19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/sasatom_i386.h
+++ b/src/sasatom_i386.h
@@ -25,6 +25,7 @@
    :
    : "memory"
   );
+}
 
 static inline void *
 __arch_fetch_and_add_ptr (void**pointer, long int delta)
